cluster_traffic_capture/cluster_traffic_capture/gen_haproxy_cfg.py

```diff
@@ -71,7 +71,9 @@ def _get_fe_config_haproxy(haproxy_port: int, mirror: bool) -> str:
     declare capture request len 80000
     declare capture response len 80000
     http-request capture req.body id 0
-    log-format Request-URI:\\ %[capture.req.uri]\\nRequest-Method:\\ %[capture.req.method]\\nRequest-Body:\\ %[capture.req.hdr(0)]\\nResponse-Body:\\ %[capture.res.hdr(0)]
+    http-request capture req.hdrs len 512
+    log-format '{{ "request": {{ "timestamp":%Ts, "uri":"%[capture.req.uri,json('utf8ps')]", "method":"%[capture.req.method,json('utf8ps')]", "body":"%[capture.req.hdr(0),json('utf8ps')]", "headers": "%hr" }}, "response":  {{"response_time_ms":%Tr, "headers":"%hs", "body":"%[capture.res.hdr(0),json('utf8ps')]", "status_code": %ST }} }}'
```

Comment:
Is there a reason you're not json-escaping the headers as well? I'm not positive whether it's necessary or not (I think it depends on what kind of characters are legal to include in headers) but I wasn't sure if there was a specific reason you're not doing it